test/misc.jl

```diff
@@ -967,12 +967,18 @@ end
 
 include("testenv.jl")
 
-
-let flags = Cmd(filter(a->!occursin("depwarn", a), collect(test_exeflags)))
-    local cmd = `$test_exename $flags --depwarn=yes deprecation_exec.jl`
-
-    if !success(pipeline(cmd; stdout=stdout, stderr=stderr))
-        error("Deprecation test failed, cmd : $cmd")
+@testset "deprecation_exec" begin
+    flags = Cmd(filter(a->!occursin("depwarn", a), collect(test_exeflags)))
+    cmd = `$test_exename $flags --depwarn=yes deprecation_exec.jl`
+    io = IOBuffer()
+    pipln = pipeline(cmd; stdout=io, stderr=io)
+    proc = run(pipln; wait = false)
```

Comment:
Can't simply do `run(pipln)`, without having to wait manually.

Reply:
If I don't do `wait = false`, then if the command fails, it throws immediately.

Reply:
https://docs.julialang.org/en/v1/base/base/#Base.ignorestatus

Reply:
For some reason, I always thought that `ignorestatus` would cause `Base.success` to return `true` even if the process failed. So that's why I avoided it. But that wouldn't really make any sense.